docs/index.md

```diff
@@ -182,6 +184,16 @@ This is an instance of Python's `datetime.timedelta`. This will be added to `dat
 
 Default is `datetime.timedelta(seconds=300)`(5 minutes).
 
+### JWT_AUDIENCE
+This is a string that will be checked against the `aud` field of the token, if present.
+
+Default is `None`(fail if `aud` present on JWT).
+
+### JWT_ISSUER
+This is a string that will be checked against the `iss` field of the token.
+
+Default is `None`(do not check `iss` on JWT).
```

Comment:
I was just trying to interpret [this code](https://github.com/jpadilla/pyjwt/blob/78742f7bee089757839921d76604b6d24d6adc32/jwt/api.py#L184) which checks the iss on the token only if `issuer` is not None:
```python
    if issuer is not None:
        if payload.get('iss') != issuer:
            raise InvalidIssuerError('Invalid issuer')
```